openjdk.build/makefile

```diff
@@ -503,10 +793,1037 @@ test.Custom:
+test.jck8b.devtools.xml_schema.nistData.union:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=xml_schema/nistData/union,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.xml_schema.sunData:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=xml_schema/sunData,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.xml_schema.boeingData:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=xml_schema/boeingData,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.jaxws:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=jaxws,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.schema2java.nisttest:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=schema2java/nisttest,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.schema2java.structures:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=schema2java/structures,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.schema_bind.bind_class :
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=schema_bind/bind_class,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.schema_bind.bind_dom:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=schema_bind/bind_dom,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.schema_bind.bind_globalBindings:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=schema_bind/bind_globalBindings,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.schema_bind.bind_property:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=schema_bind/bind_property,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.schema_bind.bind_factoryMethod:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=schema_bind/bind_factoryMethod,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.schema_bind.bind_inlineBinaryData:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=schema_bind/bind_inlineBinaryData,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.schema_bind.bind_javaType:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=schema_bind/bind_javaType,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+test.jck8b.devtools.schema_bind.bind_schemaBindings:
+	echo Running target $@
+	$(STF_COMMAND) -test=Jck -test-args="tests=schema_bind/bind_schemaBindings,jckversion=jck8b,testsuite=DEVTOOLS" $(LOG)
+	echo Target $@ completed
+
+
 help:
```

Comment:
It would be preferable to split out the JCK help text from the rest. Bear in mind that almost nobody has access to the JCK and this has resulted in most of the JCK output being related to the JCK (and the stiuff most people will want is at the top so will have scrolled off the screen) I would suggest a "make.helpjck" target with the detailed information and a reference to that target in the "make help" info.

Reply:
Done except that I called the target help.jck to be similar to the test.xxxx target standard